plugins/woocommerce/tests/api-core-tests/README.md

```diff
@@ -127,25 +129,279 @@ After you run a test, it's best to restart the environment to start from a fresh
+
+
 ### Creating test structure
 
-The structure of the test serves as a skeleton for the test itself. You can turn it into a test by using `describe()` and `it()` methods of Playwright:
+The structure of the test serves as a skeleton for the test itself. 
 
-- [`test.describe()`](https://playwright.dev/docs/api/class-test#test-describe) - creates a block that groups together several related tests;
+Each test file requires the `@playwright/test` module to be imported as follows:
+```js
+const { test, expect } = require( '@playwright/test' );
+```
+You can create a test by using the `test.describe()` and `test()` methods of Playwright:
+
+- [`test.describe()`](https://playwright.dev/docs/api/class-test#test-describe-1) - creates a block that groups together several related tests;
 - [`test()`](https://playwright.dev/docs/api/class-test#test-call) - actual method that runs the test.
 
 Based on our example, the test skeleton would look as follows:
 
 ```js
-test.describe( 'Merchant can create virtual product', () => {
-	test( 'merchant can log in', async () => { } );
+test.describe( 'Coupons API tests', () => {
+	test( 'can create a coupon', async ( {request} ) => {
+    // test to create a coupon here
+  } );
+
+	test( 'can retrieve a coupon', async ( {request} ) => {
+    // test to retrieve a coupon here
+  } );
+
+	test( 'can update a coupon', async ( { request } )  => {
+    // test to update a coupon here
+  } );
+} );
+```
 
-	test( 'merchant can create virtual product', async () => { } );
+Note: you can also nest a `test.describe()` inside a `test.describe()`. Example:
 
-	test( 'merchant can verify that virtual product was created', async () => { } );
-} );
+```js
+test.describe('Orders API tests: CRUD', () => {
+	let orderId; //test variable
+
+	test.describe('Create an order', () => {
+		test('can create a pending order by default', async ({request}) => {
+      //test code here
+    }
+```
+This allows you to further subgroup tests. When viewing the tests results locally, each test describe 'level' will be separated by `>` in the console as below: 
+
+`Orders API tests: CRUD › Create an order › can create a pending order by default`
+
+## Test Data Setup/Teardown
+
+You may need test data setup prior to the execution of your tests and removed after the execution of your tests. This can be achieved with any of the following methods, depending on the needs of the test:
+
+- [`test.beforeAll()`](https://playwright.dev/docs/api/class-test#test-before-all) - runs before all the tests in file/group
+- [`test.afterAll()`](https://playwright.dev/docs/api/class-test#test-after-all) - runs after all the tests in file/group
+- [`test.beforeEach()`](https://playwright.dev/docs/api/class-test#test-before-each) - runs before each test in file/group
+- [`test.afterEach()`](https://playwright.dev/docs/api/class-test#test-after-each) - runs after each test in file/group
```

Comment:
Would it be worth clarifying that the order execution is this or perhaps it would be too redundant?:
1. test.beforeAll()
2. test.beforeEach()
3. test.afterEach()
4. test.afterAll()

Reply:
I think it is worth re-ordering for emphasis